NorthstarDLL/squirrel.h

```diff
@@ -237,11 +237,6 @@ class SquirrelManagerBase
 		return __sq_setuserdatatypeid(sqvm, stackpos, typeId);
 	}
 
-	template <typename T> inline SQBool getthisentity(HSquirrelVM* sqvm, T* ppEntity)
-	{
-		return __sq_getentity(sqvm, (void**)ppEntity);
-	}
-
```

Comment:
Why is this removed?

Reply:
`__sq_getentity` is never declared or defined in the project. From what I can tell, MSVC doesn't care because `getthisentity` is never used so it never evaluated the template. gcc is evaluating the template regardless and doesn't compile.